### DISCUSSION TITLE
Demande de push

### DISCUSSION THREAD

**Discussion by trenaudie** [boisgera/poisson-image-editing#7] [General]
Bonjour monsieur,
Nous avons avancé sur le projet et voudrions maintenant push vers ce github. 
Pourriez-vous me donner l'autorisation? 
Merci beaucoup.
Cordialement,
Tanguy

**Comment by boisgera** [comment]
Je vous ai envoyé des invitations il y a plus de deux mois (à toi ainsi qu'à mehmet, je n'ai pas l'identifiant github du 4eme) mais elles ont depuis expiré. Je viens de recommencer ; vérifiez votre spam le cas échéant svp.
